src/pages/overlay/components/overlay/Overlay.tsx

```diff
@@ -89,6 +89,16 @@ export default function Overlay() {
   const timeoutRef = useRef<NodeJS.Timeout | undefined>(undefined);
   const awakingRef = useRef(false);
 
+  // update setting when opened menu changes
+  useEffect(() => {
+    settings.openedMenu.change(visibleOption);
+  }, [visibleOption]);
+
+  // open saved (or default) menu when mounted
+  useEffect(() => {
+    setVisibleOption(settings.openedMenu.value);
+  }, []);
```

Comment:
```suggestion
  }, [settings.openedMenu.value]);
```
It should be safe to have `settings.openedMenu.value` as a dependency here, to keep eslint happy?

Reply:
That would create a cycle since I'm updating the setting in the previous `useEffect` which depends on the state. I'm not seeing the `exhaustive-deps` rule in the project config though, but I can add an ignore if necessary.

Reply:
🤔 Maybe I'm missing something, but it shouldn't create a cycle because they should be in sync? You set the setting to `a` in the previous effect, this effect runs and sets the visible option to `a`, at which point the previous effect won't run again because `a === a`?

Reply:
Oh, I was initializing the state to an empty string, fixed.